crates/codegen/language/definition/src/model/item.rs

```diff
@@ -8,18 +10,18 @@ use crate::model::{
 };
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
-#[derive_spanned_type(EnumDiscriminants, ParseInputTokens, WriteOutputTokens)]
+#[derive_spanned_type(Clone, Debug, EnumDiscriminants, ParseInputTokens, WriteOutputTokens)]
 pub enum Item {
-    Struct { item: StructItem },
-    Enum { item: EnumItem },
-    Repeated { item: RepeatedItem },
-    Separated { item: SeparatedItem },
-    Precedence { item: PrecedenceItem },
+    Struct { item: Rc<StructItem> },
```

Comment:
Strongly agree. The tuple form is what we use elsewhere.

Reply:
Not sure I agree here. The tuple access syntax `.0` everywhere makes it for me less readable. It doesn't tell me what it holds, or if there is a `.1` or a `.2` I should also think about. It is fine in a single module for a private type that won't be used outside the file, but not for the most commonly used and shared data types across the codebase (the DSL model).

I hear you on `item.item` readability. However, I think the solution is to improve the enum field names here; Using something more descriptive than `.item`. I don't think `.0` makes it better here. Happy to hear your suggestions?

Reply:
But we have for example
```
#[derive(Clone)]
pub enum GrammarElement {
    ScannerDefinition(ScannerDefinitionRef),
    TriviaParserDefinition(TriviaParserDefinitionRef),
    ParserDefinition(ParserDefinitionRef),
    PrecedenceParserDefinition(PrecedenceParserDefinitionRef),
}
```
because these are always destructured in matches, so you never use `.0`.

Reply:
I see. In that case, that sounds good to me. Thank you.
I will add a TODO for myself to do that rename and update the parser later.